### PR TITLE
Port TOR guide from WIKI

### PR DESCRIPTION
**Pull request by Toakan**
This ports the TOR general usage and setup guide from the WIKI, but fixes some Markdown issues and separates it into readable chunks.

Source: https://github.com/pi-hole/pi-hole/wiki/DNS-over-Tor

Ran through the mkdocs serv command to check accuracy.

image

Signed-off-by: Tom Daykin <tdaykin@live.co.uk>

### CODE REVIEW

File: docs/guides/tor/overview.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+**This is an unsupported configuration created by the community**
```
Comment (Mcat12) [inline docs/guides/tor/overview.md]: Please put this message at the top of every page in this section.

File: docs/guides/tor/overview.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+If you want to protect your - unencrypted by default - DNS requests from easily being collected by your ISP or another Adversary between you and your DNS server, you can easily setup pihole to use [Tor](https://www.torproject.org) for hostname resolving. Using DNS over Tor anonymizes your IP by using [Onion-Routing](https://en.wikipedia.org/wiki/Onion_routing).
+
+#### Contribute to the Tor project  
+ If you got spare resources consider [running a Tor Relay](https://www.torproject.org/docs/tor-doc-relay.html.en) (or [Exit](https://blog.torproject.org/tips-running-exit-node)) Node to contribute back to the Tor Network. The default installation doesn't do either of these. And/Or consider [donating](https://donate.torproject.org).
+\ No newline at end of file
```
Comment (Mcat12) [inline docs/guides/tor/overview.md]: Spurious line here.

File: docs/guides/tor/using-tor.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+ To enhance your privacy you might want to route all or part of your Browser Traffic over Tor.
+ ## Tor Browser
```
Comment (Mcat12) [inline docs/guides/tor/using-tor.md]: This header does not render

File: docs/guides/tor/using-tor.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+The easiest and most reliable solution would be to use the [Tor Browser](https://www.torproject.org/projects/torbrowser.html.en). Though that won't use your pihole DNS Server out of the box. You can however disable `Proxy DNS when using SOCKS v5` in Tor Browsers Preferences -> Advanced -> Network -> Settings and [make sure](#make-sure-it-works) to point your system to use pihole with DNS over Tor activated.
+ ## Your Browser
```
Comment (Mcat12) [inline docs/guides/tor/using-tor.md]: This header does not render

**Comment by Toakan** [comment]
Changes as requested @Mcat12.

On your side note, do you mean moving the "Getting started" and "FTLDNS" nav's into their own sub category? As is, there is not too many top level options, but if more get added it would help clean up.

My own aside, https://pi-hole.net/ doesn't have a link to the Docs on the main Nav.

note: Just because I keep having this, i'll note `sed -i -e 's/pihole/Pi-hole/g' ./*` the command to tidy up for future reference.

**Comment by Mcat12** [comment]
I forwarded on the docs link request, so it should be added soon.

About the community vs official guides, I meant move the guides which currently have the "unsupported" disclaimer into a separate category under Guides possibly called "Community Guides". This does not have to be done in this PR though.

**Pull request merged**